data/categories.txt

```diff
@@ -7366,6 +7367,17 @@ el:Βουλκανιζατέρ|κατάστημα ελαστικών
 sk:Obchod s pneumatikami
 fa:فروشگاه لاستیک
 
+shop-haberdashery
+en:Haberdashey|Sewing shop
+ru:Галантерея
```

Comment:
1. The translation is incorrect.
2. Adding new types in OM is not that simple, unfortunately. They should be declared in the generator first. Please create an issue so @vng can do it. It would be great to also suggest some good icons from our listed sources (data/copyright.html)

I would also suggest avoiding creating two similar types, merging them at the generation time, and always use shop-sewing in UI and the editor.

Reply:
Ok, I created the issue: #2017